src/Components/main.css

```
@@ -28,8 +28,9 @@ body{
 
 .row_space{
     display: flex;
-    justify-content: space-evenly;
-    margin: 30px 0;
+    flex-direction: column;
+    justify-content: space-around;
+    margin: 30px 30px;
```

Just a general note, but you can use shorthand CSS here to set `30px` on all sides.

```suggestion
    margin: 30px;
```